README.md

```diff
@@ -36,4 +36,68 @@ To run all microservices docker containers, simply run the following command:
 
 To stop all containers, run the following:
 
-` docker-compose stop `
\ No newline at end of file
+` docker-compose stop `
+
+## Microservice Database Creation
+
+### Prerequisites
+* A MySQL 5.6.x server
+* An account on the MySQL server that has sufficient privileges to create database schemas, users and database objects
+
+### Create Database Schemas for Microservices
+To create the databases that store data for the microservices (`exam` and `exam_audit`), take the following steps:
+* Navigate to `database/exam` in this repository
+* Open the `V3__exam_create_application_users.sql`
+* Update the password for the **tds-app-user** and **tds-read-only** user to a value suitable for your organization
+* Execute `V1__exam_create_schema.sql` on the database server that will host the `exam` database
+* Execute `V2__exam_audit_create_schema.sql` on the database server that will host the `exam_audit` databasee
+  * **NOTE:** the `exam_audit` database does not have to exist on the same database server as the `exam` database
+
+### Create Application User Accounts for Microservices
+The following MySQL user accounts are used by the TDS microservices:
+
+* **tds-app-user:** A user account that has read/write permission in the `exam` and `exam_audit` databases
+* **tds-read-only:** A user acount taht has read permission in the `exam` database
+
+Execute the following SQL to create the MySQL user accounts with appropriate access to the `exam` database:
+
```

Comment:
A note about the create user stuff:

Since the `exam` and `exam_audit` databases can technically be hosted anywhere, it was difficult to come up with a good way to write a script for creating the application users.  MySQL 5.6 doesn't support `CREATE USER IF NOT EXISTS`; the `IF NOT EXISTS` for user creation is only supported in 5.7 and beyond.